CHANGES.md

```diff
@@ -7,6 +7,7 @@ number or not.
 ## Gameplay and UI
 
 - Default teams now have logos
+- International players from 85 countries, and better names for American players
```

Comment:
Yes, but it shouldn't be showing up in this PR because it's already in dumbmatter:master. That's why I'm confused.